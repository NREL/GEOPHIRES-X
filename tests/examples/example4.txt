----------------------------------------------------------------------------------------
GEOPHIRES v2.0 Input File
Geothermal Electricity Example Problem using Percentage Thermal Drawdown Model (Example 4)
Created by NREL on 3/26/2018
Last modified on 6/12/2018

Example 4 Description: This example problem considers a simple hydrothermal reservoir
at 2 km depth with an initial production temperature of 145deg.C. The thermal drawdown
is assumed linear at 0.5%/year. The heat is converted to electricity with a subcritical
ORC.
----------------------------------------------------------------------------------------

*** Subsurface technical parameters ***
****************************************
Reservoir Model,4,			--- Percentage thermal drawdown model
Drawdown Parameter,0.005, 		--- In [1/year].
Reservoir Depth,2, 			--- [km]
Number of Segments,1,			--- [-]
Gradient 1,65, 				--- [deg.C/km]
Number of Production Wells,3,		--- [-]
Number of Injection Wells,2,		--- [-]
Production Well Diameter,9.625,		--- [inch]
Injection Well Diameter,9.625,		--- [inch]
Ramey Production Wellbore Model,0,	--- Should be 0 (disable) or 1 (enable)
Production Wellbore Temperature Drop,0,	--- [deg.C]
Injection Wellbore Temperature Gain,0,	--- [deg.C]
Production Flow Rate per Well,110, 	--- [kg/s]
Maximum Temperature,375,		--- [deg.C]
Reservoir Volume Option,4,		--- Should be 1 2 3 or 4. See manual for details.
Reservoir Volume,1e9,			--- [m3] (required for reservoir volume option 3 and 4
Water Loss Fraction,0.0,		--- [-] (total geofluid lost)/(total geofluid produced)
Injectivity Index,10,			--- [kg/s/bar]
Productivity Index,10,			--- [kg/s/bar]
Injection Temperature,70,		--- [deg.C]
Maximum Drawdown,1,			--- [-] Maximum allowable drawdown before redrilling (a value of 0.1 means redrilling of 10% drawdown)
Reservoir Heat Capacity,1050,		--- [J/kg/K]
Reservoir Density,2700,			--- [kg/m3]
Reservoir Thermal Conductivity,3,	--- [W/m/K]


*** Surface technical parameters ***
************************************
End-Use Option,1,			--- [-] Electricity
Power Plant Type,1,			--- [1] Subcritical ORC
Circulation Pump Efficiency,0.8,	--- [-]
Utilization Factor,0.9,			--- [-]
Surface Temperature,15,			--- [deg.C]
Ambient Temperature,15,			--- [deg.C]

*** Economic/Financial Parameters ***
*************************************
Plant Lifetime,30,			--- [years]
Economic Model,3,			--- Should be 1 (FCR model) 2 (Standard LCOE/LCOH model) or 3 (Bicycle model).
Fraction of Investment in Bonds,0.65,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflated Bond Interest Rate,0.07,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflated Equity Interest Rate,0.12,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflation Rate,0.025,			--- [-] Required if Bicycle model is selected. See manual for details.
Combined Income Tax Rate,0.392,		--- [-] Required if Bicycle model is selected. See manual for details.
Gross Revenue Tax Rate,0,		--- [-] Required if Bicycle model is selected. See manual for details.
Investment Tax Credit Rate,0,		--- [-] Required if Bicycle model is selected. See manual for details.
Property Tax Rate,0,			--- [-] Required if Bicycle model is selected. See manual for details.
Inflation Rate During Construction,0,	--- [-]
Well Drilling and Completion Capital Cost Adjustment Factor,1,	--- [-] Use built-in well cost correlation as is
Well Drilling Cost Correlation,1,				--- [-] Use built-in well drilling cost correlation #1
Reservoir Stimulation Capital Cost,0,				--- [M$/injection well] Reservoir stimulation capital cost per injection well
Surface Plant Capital Cost Adjustment Factor,1,			--- [-] Use built-in surface plant cost correlation as is
Field Gathering System Capital Cost Adjustment Factor,1,		--- [-] Use built-in pipeline distribution cost correlation as is
Exploration Capital Cost Adjustment Factor,1,			--- [-] Use built-in exploration cost correlation as is
Wellfield O&M Cost Adjustment Factor,1,				--- [-] Use built-in wellfield O&M cost correlation as is
Water Cost Adjustment Factor,1,					--- [-] Use built-in water cost correlation as is
Surface Plant O&M Cost Adjustment Factor,1,			--- [-] Use built-in surface plant O&M cost correlation as is


*** Simulation Parameters ***
Print Output to Console,1,		--- [-] Should be 0 (don't print results to console) or 1 (print results to console)
Time steps per year,4,			--- [1/year]
