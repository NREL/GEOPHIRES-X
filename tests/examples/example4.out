                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.2.11
 GEOPHIRES Build Date: 2022-06-30
 Simulation Date: 2024-01-18
 Simulation Time:  10:36
 Calculation Time:      0.010 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     6.96 MW
      Electricity breakeven price:                           11.34 cents/kWh
      Number of production wells:                             3
      Number of injection wells:                              2
      Flowrate per production well:                         110.0 kg/sec
      Well depth (or total length, if not vertical):          2.0 kilometer
      Geothermal gradient:                                    0.0650 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model  = BICYCLE
      Accrued financing during construction:                  0.00
      Project lifetime:                                      30 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                         -37.27 MUSD
      Project IRR:                                          -0.03 %
      Project VIR=PI=PIR:                                    0.33
      Project MOIC:                                         -0.18

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             3
      Number of Injection Wells:                              2
      Well depth (or total length, if not vertical):          2.0 kilometer
      Water loss rate:                                        0.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 70.0 degC
      User-provided production well temperature drop
      Constant production well temperature drop:              0.0 degC
      Flowrate per production well:                         110.0 kg/sec
      Injection well casing ID:                               0.006 meter
      Production well casing ID:                              0.006 meter
      Number of times redrilling:                             0
      Power plant type:                                       Subcritical ORC


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        375.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0650 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Annual Percentage Thermal Drawdown Model
      Annual Thermal Drawdown:                                0.500 1/year
      Bottom-hole temperature:                              145.00 degC
      Warning: the reservoir dimensions and thermo-physical properties
               listed below are default values if not provided by the user.
               They are only used for calculating remaining heat content.
      Reservoir volume provided as input
      Reservoir volume:                              1000000000 m**3
      Reservoir hydrostatic pressure:                       100.00 kPa
      Plant outlet pressure:                                688.34 kPa
      Production wellhead pressure:                         757.29 kPa
      Productivity Index:                                    10.00 kg/sec/bar
      Injectivity Index:                                     10.00 kg/sec/bar
      Reservoir density:                                   2700.00 kg/m**3
      Reservoir heat capacity:                             1050.00 J/kg/kelvin


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       145.0 degC
      Average Production Temperature:                       139.4 degC
      Minimum Production Temperature:                       133.8 degC
      Initial Production Temperature:                       145.0 degC
      Average Reservoir Heat Extraction:                     96.66 MW
      Wellbore Heat Transmission Model = Constant Temperature Drop:       0.0 degC
      Average Injection Well Pump Pressure Drop:           1666.4 kPa
      Average Production Well Pump Pressure Drop:          1448.8 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      16.43 MUSD
         Drilling and completion costs per well:              3.29 MUSD
         Stimulation costs:                                   0.00 MUSD
         Surface power plant costs:                          31.58 MUSD
         Field gathering system costs:                        3.71 MUSD
         Total surface equipment costs:                      35.29 MUSD
         Exploration costs:                                   3.71 MUSD
      Total capital costs:                                   55.43 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.48 MUSD/yr
         Power plant maintenance costs:                       1.31 MUSD/yr
         Water costs:                                         0.00 MUSD/yr
      Total operating and maintenance costs:                  1.80 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.10 MW/(kg/s)
      Maximum Total Electricity Generation:                   9.46 MW
      Average Total Electricity Generation:                   8.31 MW
      Minimum Total Electricity Generation:                   7.23 MW
      Initial Total Electricity Generation:                   9.46 MW
      Maximum Net Electricity Generation:                     8.15 MW
      Average Net Electricity Generation:                     6.96 MW
      Minimum Net Electricity Generation:                     5.83 MW
      Initial Net Electricity Generation:                     8.15 MW
      Average Annual Total Electricity Generation:           64.97 GWh
      Average Annual Net Electricity Generation:             54.40 GWh
      Initial pumping power/net installed power:            100.00 %
      Average Pumping Power:                                  1.35 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                145.00               1.3117            8.1455                7.7950
   2           0.9974                144.62               1.3145            8.0615                7.7536
   3           0.9948                144.24               1.3173            7.9778                7.7123
   4           0.9922                143.87               1.3200            7.8947                7.6710
   5           0.9896                143.49               1.3228            7.8120                7.6297
   6           0.9870                143.11               1.3255            7.7297                7.5884
   7           0.9844                142.73               1.3283            7.6479                7.5471
   8           0.9817                142.35               1.3310            7.5665                7.5058
   9           0.9791                141.97               1.3337            7.4856                7.4646
  10           0.9765                141.60               1.3364            7.4052                7.4234
  11           0.9739                141.22               1.3391            7.3252                7.3821
  12           0.9713                140.84               1.3418            7.2456                7.3409
  13           0.9687                140.46               1.3445            7.1665                7.2997
  14           0.9661                140.08               1.3472            7.0878                7.2585
  15           0.9635                139.71               1.3498            7.0096                7.2174
  16           0.9609                139.33               1.3525            6.9318                7.1762
  17           0.9583                138.95               1.3552            6.8544                7.1350
  18           0.9557                138.57               1.3578            6.7775                7.0939
  19           0.9531                138.19               1.3604            6.7010                7.0527
  20           0.9504                137.82               1.3631            6.6250                7.0116
  21           0.9478                137.44               1.3657            6.5494                6.9704
  22           0.9452                137.06               1.3683            6.4742                6.9293
  23           0.9426                136.68               1.3709            6.3995                6.8881
  24           0.9400                136.30               1.3735            6.3252                6.8470
  25           0.9374                135.92               1.3761            6.2513                6.8058
  26           0.9348                135.55               1.3787            6.1778                6.7647
  27           0.9322                135.17               1.3812            6.1048                6.7235
  28           0.9296                134.79               1.3838            6.0322                6.6823
  29           0.9270                134.41               1.3863            5.9601                6.6412
  30           0.9244                134.03               1.3889            5.8883                6.6000


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  63.9                       821.8                209.67                   1.39
   2                  63.2                       817.6                206.72                   2.78
   3                  62.6                       813.5                203.79                   4.15
   4                  61.9                       809.3                200.88                   5.52
   5                  61.3                       805.2                197.98                   6.89
   6                  60.6                       801.0                195.10                   8.24
   7                  60.0                       796.9                192.23                   9.59
   8                  59.3                       792.7                189.38                  10.93
   9                  58.7                       788.5                186.54                  12.27
  10                  58.1                       784.4                183.71                  13.60
  11                  57.4                       780.2                180.91                  14.92
  12                  56.8                       776.1                178.11                  16.23
  13                  56.2                       771.9                175.33                  17.54
  14                  55.6                       767.8                172.57                  18.84
  15                  55.0                       763.6                169.82                  20.13
  16                  54.3                       759.5                167.09                  21.42
  17                  53.7                       755.3                164.37                  22.70
  18                  53.1                       751.2                161.66                  23.97
  19                  52.5                       747.0                158.97                  25.23
  20                  51.9                       742.9                156.30                  26.49
  21                  51.3                       738.7                153.64                  27.74
  22                  50.7                       734.5                150.99                  28.99
  23                  50.2                       730.4                148.37                  30.22
  24                  49.6                       726.2                145.75                  31.45
  25                  49.0                       722.1                143.15                  32.67
  26                  48.4                       717.9                140.57                  33.89
  27                  47.8                       713.8                138.00                  35.10
  28                  47.3                       709.6                135.44                  36.30
  29                  46.7                       705.5                132.90                  37.49
  30                  34.7                       526.4                131.01                  38.39
